pubtools/_quay/push_docker.py

```diff
@@ -495,6 +499,10 @@ def remove_old_signatures(
                 claim messages created for new index image(s)
 
         """
+        # if there are no rollback tags, it means content is repushed. And therefore nothing
+        # should be removed.
```

Comment:
It's trying to change the condition of removing signatures from `when tags are overwritten with new content` to `when there's no new tag`. It still doesn't make sense to me.

Reply:
ah, I was doing this change quickly between migration chunks yesterday. Obviously there should be 
if not rollback_tags. Basically if there are no new tags, it means it's a repush and signatures shouldn't be removed